handbook/support/p4-enablement.md

```diff
@@ -0,0 +1,163 @@
+# Dogfood Perforce Server
+
+Perforce is a version control system like git, subversion, or mercurial. While git is based on a distributed, decentralised model, Perforce is centralised. For testing purposes you may use our perforce dogfood server.
+
+## Setting Up
+- To connect to perforce you'll need the perforce cli installed locally. Use the command: `brew install --cask perforce`
+- The following environment variables configure your shell to point at the perforce server. Set them to your `env` with the `export` command, or add them to your `.bashrc` or `.zshrc` file.
+```
+# .zshrc or .bashrc
+
+P4PORT=perforce.sgdev.org:1666 # This points the p4 cli at the dogfood server
+P4USER=admin # sets your user
+P4PASSWD=<login session ticket value explained below> # doesn't require string quotes
+P4EDITOR=/usr/bin/vim # specifies the editor opened by some p4 commands
+```
+## Interacting With Perforce Dogfood
+
+To add repos to the perforce dogfood server follow the following procedure:
+- [Dogfood Perforce Server](#dogfood-perforce-server)
+  - [Setting Up](#setting-up)
+  - [Interacting With Perforce Dogfood](#interacting-with-perforce-dogfood)
+    - [Generate A Session Ticket](#generate-a-session-ticket)
+    - [Create New Depot](#create-new-depot)
+    - [Create A Client](#create-a-client)
+    - [Add Files and Submit](#add-files-and-submit)
+    - [Configuring Sourcegraph To Sync Dogfood Depots](#configuring-sourcegraph-to-sync-dogfood-depots)
+- [Local Perforce Server](#local-perforce-server)
+
+### Generate A Session Ticket
+Interaction with the dogfood server requires authentication. Once you've set up your shell environment you'll need to generate a session ticket. To do this you'll need your users password. You can find the [admin password](https://team-sourcegraph.1password.com/vaults/dnrhbauihkhjs5ag6vszsme45a/allitems/fac6hoq3ujb3xpxtllbijzyxta), and others on our shared 1Password account.
+
+1. Once your environment is set run `p4 ping` this will prompt you for the admin password, and is a good way to test your connection to the server.
```

Comment:
If you have configured `P4PASSWD` correctly, `p4` will not ask for a password again and just take it. However, by default session ticket has a lifetime of 12 hours, so in the interactive mode (ie. terminal), `P4PASSWD` is actually not as useful/needed.

Reply:
🤔  I'm not sure I understand this, my work flow involved setting the session ticket to the env variable P4PASSWD, which doesn't require updating for like 12 hours as you've noted. If I didn't most p4 commands would regularly require entering the admin password. Is there a more efficient way to do this I've overlooked? 

I think I can clarify whats already there a bit more though 👍

Reply:
By default, every time you entered a password interactively, a session ticket is stored directly in your home directory `~/.p4tickets`, which is the same value as you would put as `P4PASSWD`.

But having an extra step does not hurt 😁

Reply:
> If I didn't most p4 commands would regularly require entering the admin password.

I think the reason for this because you do have `P4PASSWD` set, either invalid or set with empty string.

Reply:
Ahh yeah you're right, I set P4PASSWD early, and only ever ran `p4 login` with the -p flag which prevents storage to `.p4tickets`

Reply:
added a note about alternative login with .p4tickets